whitepapers/0013_ccq.md

```diff
@@ -289,6 +295,35 @@ Currently the only supported query types are `eth_call` and `eth_call_by_timesta
    []byte   batch_call_data
    ```
 
+#### Solana Queries
+
+Currently the only supported query type on Solana is `sol_account`.
+
+1. sol_account (query type 10) - this query is used to read data for one or more accounts on Solana.
+
+   ```go
+   u32      commitment_len
+   []byte   commitment
+   u64      min_context_slot
+   u64      data_slice_offset
+   u64      data_slice_length
+   u8       num_accounts
+   []byte   account_list
```

Comment:
can account list just be `[]byte32` and specified as the bytes instead of the base58 string?